charts/pega/templates/pega-environment-config.yaml

```diff
@@ -54,5 +54,14 @@ data:
   PEGA_STREAM_URL: {{ .Values.stream.url }}
 {{- end }}
 {{- end }}
-
+{{ if .Values.hazelcast.enabled }}
+  HZ_DISCOVERY_K8S: "true"
+  HZ_CLIENT_MODE: "true"
+{{ if .Values.hazelcast.client.clusterName }}
+  # Hazelcast Cluster Name
+  HZ_CLUSTER_NAME: {{ .Values.hazelcast.client.clusterName }}
+{{- end }}
+  # DNS of hazelcast service
+  HZ_DNS: {{ template "hazelcastName" . }}-service.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
DNS is not clear in this context.  Maybe HZ_SERVER_HOSTNAME would be better.

Reply:
As per suggestions from Dave and you , I am going with this name:-
HZ_SERVER_URL